### PR TITLE
Add a comment explaining the necessary scale for Density test

### PR DESCRIPTION
**Pull request by gmarek**
cc @yujuhong

### CODE REVIEW

File: test/e2e/density.go [hunk 1]
```diff
@@ -113,6 +113,9 @@ func density30AddonResourceVerifier() map[string]resourceConstraint {
 // So by default it is added to the ginkgo.skip list (see driver.go).
 // To run this suite you must explicitly ask for it by setting the
 // -t/--test flag or ginkgo.focus flag.
+// IMPORTANT: This test is designed to work on large (>= 100 Nodes) clusters. For smaller ones
+// results will not be representative for control-plane performance as we'll start hitting
+// limits on Docker's concurrent container startup.
```
Comment (yujuhong) [inline test/e2e/density.go]: Hmm.....the qps limit of the apiserver client is more of a bottleneck now than docker during batch creation.

Comment (gmarek) [inline test/e2e/density.go]: How many QPS does Kubelet client allow?

Comment (yujuhong) [inline test/e2e/density.go]: 5.0.

Some numbers:
with 5 qps, starting 100 pods per node takes 1m50s to 2m20s
with 100qps, starting 100 pods per node takes ~1m20s

Comment (gmarek) [inline test/e2e/density.go]: @wojtek-t - we probably should increase this.

Comment (wojtek-t) [inline test/e2e/density.go]: 100 is too much in my opinion
But probably increasing to 10 or 20 should be ok - we should make some experiments.

**Pull request merged**